test/Microsoft.AspNet.Mvc.FunctionalTests/ModelBindingTest.cs

```diff
@@ -2067,6 +2067,23 @@ public class ModelBindingTest
             Assert.False(result[1].Checked);
         }
 
+        [Fact]
+        public async Task TryUpdateModel_InvalidModelAsParameter_FixOnServer_ReturnsEmptyResponseBody()
+        {
+            // Arrange
+            var server = TestServer.Create(_services, _app);
+            var client = server.CreateClient();
+            var url = "http://localhost/TryUpdateModel/TryUpdateModel_ClearsModelStateEntries";
+
+            // Act
+            var response = await client.GetAsync(url);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            var body = await response.Content.ReadAsStringAsync();
+            Assert.Equal("{}", body);
```

Comment:
emtpy string as json??? that shouldnt be the case, it should have been string.Empty